[Marketplace Contribution] Microsoft Graph Device Management - Content Pack Update (#18786)
demisto/content: Packs/MicrosoftGraphDeviceManagement/ReleaseNotes/1_0_16.md

#### Integrations
##### Microsoft Endpoint Manager (Intune)

> New command "msgraph-find-managed-device-by-name" is added. It allows to search of managed device by its name.

Reviewer: ```suggestion
- Added the ***msgraph-find-managed-device-by-name*** command.
  It searches a managed device by its name.
```
